adapters/pubmatic/pubmatic.go

```diff
@@ -465,6 +476,20 @@ func (a *PubmaticAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externa
 	if bidResp.Cur != "" {
 		bidResponse.Currency = bidResp.Cur
 	}
+
+	if bidResp.Ext != nil {
+		var bidRespExt respExt
+		if err := json.Unmarshal(bidResp.Ext, &bidRespExt); err == nil && bidRespExt.FledgeAuctionConfigs != nil {
```

Comment:
It is unsafe to ignore json Unmarshal error here. Requesting you to please add Unmarshal error check.

Reply:
We don't want to reject the whole response due to unmarshal error of fledge object. In case of error, the adapter would drop the fledge response/object and continue with the regular bid response if any.

Reply:
@gargcreation1992  could you please address the above comment?